modules/tinymce/src/themes/silver/main/ts/modes/ScrollingContext.ts

```diff
@@ -14,12 +14,23 @@ export interface ScrollingContext {
 // TINY-9385: Investigate exactly what makes an element possibly
 // scrollable. This approach does not seem exhaustive. What about:
 // overflow-x, overflow-y etc.
-const nonScrollingOverflows = [ 'visible', 'hidden' ];
+const nonScrollingOverflows = [ 'visible', 'hidden', 'clip' ];
 
 export const isScroller = (elem: SugarElement<Node> | any): boolean => {
   if (SugarNode.isHTMLElement(elem)) {
     const overflow = Css.get(elem, 'overflow');
-    return Strings.trim(overflow).length > 0 && !Arr.contains(nonScrollingOverflows, overflow);
+    const overflowX = Css.get(elem, 'overflow-x');
+    const overflowY = Css.get(elem, 'overflow-y');
+
+    // If overflow-x and overflow-y have different values, ignore 'overflow'
+    if (overflowX !== overflowY) {
+      return (Strings.trim(overflowX).length > 0 && !Arr.contains(nonScrollingOverflows, overflowX)) ||
+        (Strings.trim(overflowY).length > 0 && !Arr.contains(nonScrollingOverflows, overflowY));
+    }
+
+    return (Strings.trim(overflow).length > 0 && !Arr.contains(nonScrollingOverflows, overflow)) ||
+      (Strings.trim(overflowX).length > 0 && !Arr.contains(nonScrollingOverflows, overflowX)) ||
+      (Strings.trim(overflowY).length > 0 && !Arr.contains(nonScrollingOverflows, overflowY));
```

Comment:
Since we are using `Css.get` which gets computed values we shouldn't need to also check `'overflow'` as when `'overflow: <value>'` is set in the DOM, the 'overflow-x' and 'overflow-y' properties are also set.

```
const isScrollingOverflowValue = (value: string): boolean =>
  Strings.trim(value).length > 0 && !Arr.contains(nonScrollingOverflows, value);
```

```suggestion
    const overflowX = Css.get(elem, 'overflow-x');
    const overflowY = Css.get(elem, 'overflow-y');
    return isScrollingOverflowValue(overflowX) || isScrollingOverflowValue(overflowY);
```